src/main/java/io/cdap/plugin/google/common/GoogleAuthBaseConfig.java

```diff
@@ -54,6 +54,9 @@ public abstract class GoogleAuthBaseConfig extends PluginConfig {
   public static final String SERVICE_ACCOUNT_FILE_PATH = "filePath";
   public static final String SERVICE_ACCOUNT_JSON = "JSON";
   public static final String SCHEMA = "schema";
+  public static final String ACCESS_TOKEN = "accessToken";
+  public static final String ACCESS_TOKEN_LABEL = "Access token";
```

Comment:
IIUC label is picked from widgets json, there it is specified as `Access Token` right?

Reply:
similar problem with `REFRESH_TOKEN_LABEL` on line 49.

Reply:
This is used to just to form error message, I changed all the labels to have uniformity.